docs/DefaultDeployment.md

```diff
@@ -68,7 +70,7 @@ The template can also be deployed using PowerShell. Here is an example of how th
 $rg = New-AzResourceGroup -Name "RG-NAME" -Location westus2
 
 New-AzResourceGroupDeployment `
--TemplateUri "https://raw.githubusercontent.com/Microsoft/fhir-server/master/samples/templates/default-azuredeploy.json" `
+-TemplateUri "https://raw.githubusercontent.com/Microsoft/fhir-server/master/samples/templates/default-azuredeploy-sql.json" `
```

Comment:
I don't think this will just work as sql requires extra/different parameters, e.g. sql server password

Reply:
I think the two that would be required would be sql server password and sql schema automatic updates enabled.

Reply:
Thanks! I reverted and just made it clear that its Cosmos